src/redux/actions/authUser.js

```diff
@@ -1,20 +1,31 @@
-// Signup user
 import axios from "axios";
 
 import { SIGNUP_SUCCESS, SIGNUP_ERROR } from "../action_types";
 import { serverError, startFetch } from "./common";
 
 export const signupSuccess = message => ({
+  /**
+   * Signup successfull
+   *
+   * @param (string) message
+   * @return (object) type and payload
+   */
   type: SIGNUP_SUCCESS,
   message
 });
 
 export const signupError = error => ({
+  /**
+   * Signup
+   *
+   * @param (object) error
+   * @return (object) type and payload
+   */
   type: SIGNUP_ERROR,
   error
 });
 
-export default user => dispatch => {
+export const signup = user => dispatch => {
```

Comment:
@kimotho-njoki Exported the function without default

Reply:
Didn't I suggest a refactoring to this? I believe I did...I even wrote it out for you.. `dispatch` should be on the next line.

Reply:
@kimotho-njoki prettier is automatically refactoring to this

Reply:
@kimotho-njoki I have refactored appropriately